.github/workflows/run_db_trigger.yml

```
@@ -19,88 +19,134 @@ on:
         description: 'S3 URI'
         required: true
         type: string
-      file_name:
-        description: 'File Name in S3'
-        required: true
-        type: string
+
+
         
     secrets:
-      DOCKER_PASSWORD:
-        required: true
-      DOCKER_USERNAME:
-        required: true
       AWS_ACCESS_KEY_ID:
         required: true
       AWS_SECRET_ACCESS_KEY:
         required: true
       SLACK_WEBHOOK_URL:
         required: true
-      
+
+
+
+env:
+  REGISTRY: ghcr.io
+    
 jobs:
-  db_image_build_push:
+  export-db:
     runs-on: ubuntu-latest
     permissions:
       contents: read
-      packages: read
-      id-token: none
-
+      packages: write
+      id-token: write
+    env:
+      REGISTRY: ghcr.io
+      IMAGE_NAME: ${{ github.repository }}/mariadb
+      TMP_DATA_DIR: data
+      DB_FILE: db.sql
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
+      - name: Start the database container
+        id: container
+        run: |
+          CONTAINER_ID=$(docker run -d ghcr.io/dpc-sdp/bay/mariadb:5.x)
+          timeout 60s docker exec --user 1000 -i "${CONTAINER_ID}" sh -c "until nc -z localhost 3306; do sleep 1; echo -n .; done; echo"
+          echo "id=$CONTAINER_ID" >> $GITHUB_OUTPUT
+
+      - name: Configure AWS actions
+        uses: aws-actions/configure-aws-credentials@v4.0.1
```

```suggestion
        uses: aws-actions/configure-aws-credentials@v4
```
Should be fine to pin this to the major